Dockerfile

```diff
@@ -1,9 +1,12 @@
 FROM nginx:1.15-alpine
 
+COPY esh /esh
 COPY nginx.main.conf /etc/nginx/nginx.conf
-COPY nginx.proxy.conf /etc/nginx/conf.d/default.conf
-COPY nginx.proxy-cached.conf /etc/nginx/conf.d/cached.conf
+COPY nginx.proxy.conf.esh /etc/nginx/conf.d/default.conf.esh
 COPY entrypoint.sh /entrypoint.sh
 
+RUN chmod +x /entrypoint.sh
```

Comment:
I don't think we need this. But you have to update the `esh` script to be executable in the repo.